articles/azure-monitor/app/tutorial-custom-metrics.md

```diff
@@ -0,0 +1,370 @@
+
+   4. Repeat adding reviews as desired to generate additional telemetry.
+
+### View logs in Application Insights
+
+1. Go to the **Application Insights** resource in the [Azure Portal](https://portal.azure.com).
+
+    ![A resource group displays with the Application Insights resource highlighted.](./media/asp-net-core/application_insights_resource_group.png "Resource Group")
+
+2. From the left menu of the Application Insights resource, select **Logs** from beneath the **Monitoring** section. In the **Tables** pane, double-click on the **customMetrics** table, located under the **Application Insights** tree. Modify the query to retrieve metrics for the **ReviewPerformed** custom named metric as follows, then select **Run** to filter the results.
+
+    ```kql
+    customMetrics 
+    | where name == "ReviewPerformed"
+    ```
+
+3. Observe the results display the rating value present in the Review and the aggregated values.
+
+4. In order to better observe the **IncludesPhoto** dimension, we can extract it into a separate variable (column) by using the following query.
+
+    ```kql
+    customMetrics 
+    | extend IncludesPhoto = tobool(customDimensions.IncludesPhoto)
+    | where name == "ReviewPerformed"
+    ```
+
+5. Since we re-used the same custom metric name has before, results with and without the custom dimension will be displayed. In order to avoid that, we will update the query to match the following one.
+
+    ```kql
+    customMetrics 
+    | extend IncludesPhoto = tobool(customDimensions.IncludesPhoto)
+    | where name == "ReviewPerformed" and isnotnull(IncludesPhoto)
+    ```
+
+### View metrics in Application Insights
+
+1. Go to the **Application Insights** resource in the [Azure Portal](https://portal.azure.com).
+
+    ![A resource group displays with the Application Insights resource highlighted.](./media/asp-net-core/application_insights_resource_group.png "Resource Group")
+
+2. From the left menu of the Application Insights resource, select **Metrics** from beneath the **Monitoring** section.
+
+3. For **Metric Namespace**, select **azure.applicationinsights**.
+
+    ![A metrics explorer with the Metric Namespace highlighted.](./media/custom-metrics/metrics-explorer-namespace.png "Metric Namespace")
+
+4. For **Metric**, select **ReviewPerformed**.
+
+    ![A metrics explorer with the Metric highlighted.](./media/custom-metrics/metrics-explorer-metric.png "Metric")
+
+5. However, you'll notice that you aren't able to split the metric by your new custom dimension, or view your custom dimension with the metrics view. Select **Apply Splitting**.
+
+    ![Splitting support](./media/custom-metrics/apply-splitting.png "Splitting")
+
+6. For the custom dimension **Values** to use, select **IncludesPhoto**.
+
+    ![Splitting using a custom dimension](./media/custom-metrics/splitting-dimension.png "Splitting dimension")
+
+## Next steps
```

Comment:
it'd be good to include more scenarios described in this doc : https://docs.microsoft.com/en-us/azure/azure-monitor/app/get-metric

Reply:
Would that create more duplication of information in multiple documents, something that was already a concern to begin with within this tutorial?

Should maybe include a reference to that document in the "Next steps" section?